doc/userguide/configuration/exception-policies.rst

```diff
@@ -16,6 +16,28 @@ simulate failures or errors and understand Suricata behavior under such conditio
 Exception Policies
 ------------------
 
+Master Switch
+~~~~~~~~~~~~~
+
+In IPS mode, the default behavior for all exception policies is to drop packets
+and/or flows. It is possible to disable this default, by setting the exception
+policies "master switch" yaml config option, to ``disabled``:
+
+::
+
+   # Define a common behavior for all exception policies
+   # the default is drop-packet/drop-flow. To fallback to old behavior (setting
+   # each of them individually, or ignoring all), disable it.
+   # Possible values: auto (drop-packet/drop-flow), perfomance (bypass),
+   # disabled
+   exception-policy-master-switch: disabled
```

Comment:
wonder if we should just accept the exception-policy values here, in addition to "auto"

Reply:
then it could also just be called `exception-policy` in the root namespace?

Reply:
> then it could also just be called `exception-policy` in the root namespace?

I'm having a bit of trouble understanding this one. 
Do you mean that then it could be something like:
`exception-policy: <mode>`?
In the yaml configuration?

Reply:
> wonder if we should just accept the exception-policy values here, in addition to "auto"

Do I interpret it right, that we'd also have just one enum for this?